templates/react/utils/itemToLinks.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import {API_PATH} from '../api/_entrypoint';
+
+export default function(items) {
+  return Array.isArray(items)
+    ? items.map(item => createLink(item))
+    : createLink(items);
+}
+
+function createLink(item) {
+  if (typeof(item) === 'string' && item.includes(API_PATH) > 0) {
```

Comment:
There will be always "something" in API_PATH unless api is served just over plain domain name.
During development I have it set automatically by generator to:
export const API_HOST = "http://192.168.1.71:81";
export const API_PATH = "/api.epp.domeny.dev/web/app.php/api";